### ISSUE TITLE
Bug in template profiles witih 'charliecloud'

### ISSUE THREAD

**Issue by KevinMenden** [nf-core/tools#984]
In the new template, the profile options lead to errors. Specifically:

```
  conda { 
    params.enable_conda = true   
    docker.enabled      = false
    singularity.enabled = false
    podman.enabled      = false
    shifter.enabled     = false
    charliecloud        = false
  }
```

`charliecloud` should be `charliecloud.enabled` for conda and podman (it already is for the other profiles). Otherwise it can result in a failure `Unknown method invocation "isEmpty" on Boolean type `

**Comment by phue** [comment]
This is fixed in 3a2c815 
I had the impression that it would silently fail without throwing an error. But it doesn't, atleast on `21.03.0-edge` it gives the error you described. That's bad, so currently conda and podman profiles are broken.

We should start testing them..

**Comment by KevinMenden** [comment]
Great, thanks! I guess we can close this then :-)
